samcli/lib/build/app_builder.py

```diff
@@ -184,16 +184,24 @@ def _get_build_graph(self) -> BuildGraph:
         return build_graph
 
     @staticmethod
-    def update_template(template_dict: Dict, original_template_path: str, built_artifacts: Dict[str, str]) -> Dict:
+    def update_template(
+        stack_path: str,
+        template_dict: Dict,
+        original_template_path: str,
+        built_artifacts: Dict[str, str],
+        stack_output_template_path_by_stack_path: Dict[str, str],
+    ) -> Dict:
         """
         Given the path to built artifacts, update the template to point appropriate resource CodeUris to the artifacts
         folder
 
         Parameters
         ----------
-        template_dict
+        stack_path: str
+        template_dict: dict
         original_template_path : str
             Path where the template file will be written to
+        stack_output_template_path_by_stack_path: Dict[str, str]
```

Comment:
Some doc will be helpful around this parameter

Reply:
updated in https://github.com/aws/aws-sam-cli/pull/2623/commits/94ff67193081484c7ecd1e4e336f205d0389eab1